synapse/handlers/federation.py

```diff
@@ -482,7 +487,7 @@ async def try_backfill(domains: List[str]) -> bool:
 
         processing_end_time = self.clock.time_msec()
         backfill_processing_before_timer.observe(
-            (processing_start_time - processing_end_time) / 1000
+            (processing_end_time - processing_start_time) / 1000
```

Comment:
Fixed the order of these so we no longer give negative times.

`end - start` is the correct way

ex. `1200 - 1000` -> 200 (positive) 🎉